src/sql/command.py

```diff
@@ -11,7 +11,7 @@
 
 class SQLPlotCommand:
     def __init__(self, magic, line) -> None:
-        self.args = parse_argstring(magic.execute, line)
+        self.args = magic.execute.parser.parse_args(shlex.split(line, posix=False))
```

Comment:
any ideas on why it works for %sql but it didn't for %sqlplot, please document the rationale as comments

Reply:
I realized I only tested `parse_argstring` from `IPython.core.magic_arguments` and not `parse.magic_args`: [fixed](https://github.com/ploomber/jupysql/pull/822/files#diff-769097f5e36be57ddec10c78df0900db8c1e254f5485bc94367aa7fcb07991c5R14)

Added comment for why it doesn't work for `%sqlcmd`: [fixed](https://github.com/ploomber/jupysql/pull/822/files#diff-2aa72d6ecafc2697ec8174353868dae816494131c70b7e48be6115fda9f61370R74)